### PR TITLE
feat(dev-server): added dev server

### PR DESCRIPTION
**Pull request by LarsDenBakker**
No global docs changes, I want to do that as a bigger MR of updating the building/development docs.

### CODE REVIEW

File: packages/dev-server/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Dev server
+
+> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/)
+
+Open Web Components provides a set of defaults, recommendations and tools to help facilitate your Web Component. Our recommendations include: developing, linting, testing, tooling, demoing, publishing and automating.
```
Comment (thepassle) [inline packages/dev-server/README.md]:
```suggestion
Open Web Components provides a set of defaults, recommendations and tools to help facilitate your web component project. Our recommendations include: developing, linting, testing, building, tooling, demoing, publishing and automating.
```

Comment (LarsDenBakker) [reply packages/dev-server/README.md]: I copied it from testing-karma. Let's do a sweep of all the packages :P

Comment (thepassle) [reply packages/dev-server/README.md]: i did it in https://github.com/open-wc/open-wc/pull/154

**Comment by LarsDenBakker** [comment]
I wanted to pick up browser sync in a separate PR. I looked into it, but it looked quite involved.

**Pull request merged**